src/items/containers/container.cpp

```diff
@@ -819,6 +819,23 @@ ContainerIterator Container::iterator() const {
 	return cit;
 }
 
+void Container::removeItem(Thing* thing) {
```

Comment:
Thing can be const

Reply:
Thanks, but don't worry, Sonar picks up these kinds of details; that's exactly what we use it for, to avoid unnecessary reviews.